src/staticanalysis/bot/static_analysis_bot/workflows/remote.py

```diff
@@ -115,47 +57,32 @@ def run(self, revision):
                 logger.info('Skip dependency not in group', task_id=dep_id)
                 continue
             try:
-                task = AnalysisTask(dep_id, tasks[dep_id])
-                logs = task.load_logs(self.queue_service)
-                if logs is None:
-                    continue
-
-                for log in logs.values():
-                    issues += self.parse_issues(task, log, revision)
+                task = self.build_task(dep_id, tasks[dep_id])
+                artifacts = task.load_artifacts(self.queue_service)
+                if artifacts is not None:
+                    task_issues = task.parse_issues(artifacts, revision)
+                    logger.info('Found {} issues'.format(len(task_issues)), task=task.name, id=task.id)
+                    issues += task_issues
             except Exception as e:
                 logger.warn('Failure during task analysis', task=settings.taskcluster.task_id, error=e)
-                raise
```

Comment:
Maybe we should keep raising, since the analysis is now very simple (just parsing some artifacts) we should never fail

Reply:
:+1: Hardcore mode.